### PR TITLE
Story Updates

### PR DESCRIPTION
**Pull request by InfamousStarFox**
Thank you Ryan Jones for finding the errors in these stories

### CODE REVIEW

File: _posts/2009-02-21-090-loder-honored-with-lifetime-acheivement-award.markdown [hunk 1]
```diff
@@ -5,31 +5,31 @@ date: 2009-02-21 02:55:07.000000000 -07:00
...
 <br/><br/>
-The requirements for the award are few, but arduous. A nominee must be a Vigil Honor member who has been recognized with the Distinguished Service Award (â€œDSAâ€) and has continued service to the organization on the national, regional or sectional level since receiving the DSA. In addition, the nominee must have been a member of the Order for no less than 25 years. The only the recipients of the award, to date, are Thomas McBride (PA), Carl M. Marchetti, M.D. (NJ), and Dabney Kennedy (TX).
+The requirements for the award are few, but arduous. A nominee must be a Vigil Honor member who has been recognized with the Distinguished Service Award (DSA) and has continued service to the organization on the national, regional or sectional level since receiving the DSA. In addition, the nominee must have been a member of the Order for no less than 25 years. The only the recipients of the award, to date, are Thomas McBride (PA), Carl M. Marchetti, M.D. (NJ), and Dabney Kennedy (TX).
 <br/><br/>
 ---
 <br/><br/>
-Del Loderâ€™s Scouting career and OA history are lengthy and distinguished.
+Del Loder™s Scouting career and OA history are lengthy and distinguished.
```
Comment (blove) [inline _posts/2009-02-21-090-loder-honored-with-lifetime-acheivement-award.markdown]: This looks like a trademark symbol to me.

**Comment by InfamousStarFox** [comment]
Gee, yes it does. :smiley:  Is Del Loder now a trademarked figure?
I'll fix that. It's amazing how corrupted symbols are rendered

**Comment by blove** [comment]
Haha - not as far as I know.

**Comment by InfamousStarFox** [comment]
Oh my, all eleven apostrophes were being rendered as trademark symbols

**Comment by blove** [comment]
Ok, looks good, thanks.

**Pull request merged**